src/components/SelectNetworkAndAccount.svelte

```diff
@@ -111,14 +111,30 @@
     }
   }
 
+  function findMatchingAccount(): Account | null {
```

Comment:
If we don't have this, the selected value isn't an absolute equal with the options and it shows a blank in dropdown when it should show the selected value.